staging/src/k8s.io/apimachinery/pkg/api/meta/default.go

```diff
@@ -38,6 +40,9 @@ func NewDefaultRESTMapperFromScheme(defaultGroupVersions []schema.GroupVersion,
 			// We check the import path because we currently stuff both "api" and "extensions" objects
 			// into the same group within Scheme since Scheme has no notion of groups yet.
 			if !strings.Contains(oType.PkgPath(), importPathPrefix) || ignoredKinds.Has(kind) {
```

Comment:
is `!strings.Contains(oType.PkgPath(), importPathPrefix)` relevant at all anymore? Scheme knows groups for ages. This line was introduced in https://github.com/kubernetes/kubernetes/pull/12001 two years ago.